pyccel/ast/core.py

```diff
@@ -2566,9 +2566,23 @@ def results(self):
 
     @property
     def body(self):
-        """ CodeBlock containing all the statements in the function """
+        """
+        CodeBlock containing all the statements in the function.
+
+        Returns a CodeBlock contains all the statements in the function.
```

Comment:
Typo: contains -> containing
I would also prefer "Return" to "Returns"

Reply:
Fixed https://github.com/pyccel/pyccel/pull/1508/commits/63d09e3bcdf9f18da4477da007cc16875c9cb92a.